evalml/data_checks/invalid_targets_data_check.py

```diff
@@ -57,7 +59,7 @@ def validate(self, X, y):
                                                                    "code": "TARGET_HAS_NULL",\
                                                                    "details": {"num_null_rows": 2, "pct_null_rows": 50}}],\
                                                        "warnings": [],\
-                                                       "actions": []}
+                                                       "actions": [{'code': 'IMPUTE_COL', 'details': {'column': None, 'impute_strategy': 'most_frequent', 'is_target': True}}]}
```

Comment:
Wanted a way to specify that we want to impute the target without relying on the name of the column